flyteplugins/go/tasks/plugins/k8s/ray/batchscheduler/config/config_test.go

```diff
@@ -0,0 +1,15 @@
+package config
```

Comment:
should we move `batchscheduler` plugin to `flyteplugins/go/tasks/plugins/k8s`? spark/dask/kubeflow plugin can use it in the future.

Reply:
Agree.